### DISCUSSION TITLE
ADO读写分离？

### DISCUSSION THREAD

**Discussion by januswx** [dotnetcore/FreeSql#878] [Q&A] [answered]
FreeSql.Ado 是否支持读写分离，例如系统中一些实时性不高的复杂SQL报表通过从库查询，实时性要求比较高的SQL报表通过主库查询，类似于FreeSql.Select<T>().Master()一样的使用方式？

**Comment by 2881099** [comment, marked as answer]
默认以

>SELECT 

开头的查询，会查从库。

想要查主库可以这样：

> /* master */ SELECT ...

**Reply by januswx** [reply]
明白了，谢谢。
